### PR TITLE
Add Mapnik

### PR DESCRIPTION
**Pull request by jamesmcclain**
https://github.com/geotrellis/geonotebook/pull/13 does not work in jupyterhub, must use jupyter: `jupyter notebook --port=8000`.

To Test
   - Make sure that all bootstrap containers are up to date
   - Build from https://github.com/jamesmcclain/geodocker-jupyter-geopyspark/tree/feature/mapnik-wip
      - `docker run -it --rm --host=net quay.io/jupyter-geopyspark:mapnik bash`
      - `jupyter notebook --port=8000` from inside of container
      - Test that KitWare notebooks work (one is now included in this repository)
   - Build from this branch
      - Use as before; confirm that everything that used to work still does

screenshot from 2017-08-18 19 52 47
screenshot from 2017-08-20 11 12 47

Fixes geodocker/geodocker-jupyter-geopyspark#18

### CODE REVIEW

**Comment by jpolchlo** [comment]
The errors in the console that you observe are, I believe, the result of a known issue with Ktile's proxy provider (OpenGeoscience/ktile#24).

**Comment by jamesmcclain** [comment]
I think that this is complete.

**Comment by jpolchlo** [comment]
Sounds fine to me. I'll be filling a couple PRs to fix up the last couple issues. Will require some changes to GeoNotebook.

-----Original Message-----

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,25 @@
-FROM quay.io/geodocker/jupyter:99fceaa
+FROM jamesmcclain/jupyter-geopyspark:base
```
Comment (echeipesh) [inline Dockerfile]: Should be pointed to `quay.io/geodocker/jupyter-geopyspark` before merge.

File: Makefile [hunk 2]
```diff
@@ -1,45 +1,21 @@
...
 
-TAG ?= 16
-IMG := quay.io/geodocker/jupyter-geopyspark
-STAGE0 := $(IMG):stage0
-STAGE1 := $(IMG):80da618
-STAGE2 := $(IMG):$(TAG)
+TAG ?= mapnik
+FAMILY := quay.io/geodocker/jupyter-geopyspark
+AWSBUILD := jamesmcclain/jupyter-geopyspark:aws-build-gdal
+BASE := jamesmcclain/jupyter-geopyspark:base
```
Comment (echeipesh) [inline Makefile]: Assigned here but not used in the file.

File: .travis.yml [hunk 3]
```diff
@@ -43,12 +43,12 @@ before_install:
 
 install:
    - pip3 install wheel
-   - docker pull jamesmcclain/jupyter-geopyspark:stage0
+   - docker pull jamesmcclain/jupyter-geopyspark:aws-build-gdal
```
Comment (echeipesh) [inline .travis.yml]: Not published from scripts but I suppose its safe to publish this once and use as resource after image rename.

Comment (jamesmcclain) [inline .travis.yml]: Yes, these will not build on Travis (lack of RAM, lack of disk)

**Comment by jamesmcclain** [comment]
Okay, thanks, merging

**Pull request merged**